addons/mail/static/tests/web/chatter_tests.js

```diff
@@ -567,7 +568,7 @@ QUnit.test(
             res_id: partnerId,
             views: [[false, "form"]],
         });
-        await click(".o_form_button_create:eq(0)");
+        await click(".o_control_panel_collapsed_create .o_form_button_create");
```

Comment:
Not clear just at name of classname `o_control_panel_collapsed_create` where it is explicitly in UI.
Also no idea where the other `o_form_button_create` may be shown.
Would have been best another parent selector, or using `{ text: "Create" }` for example?

Reply:
Not possible because form view has template with d-none, d-xl-none or something like that with both on the template always at the same time and only one that is actually visible.